readme.md

```diff
@@ -20,6 +20,10 @@ fileUrl('unicorn.jpg');
 
 fileUrl('/Users/pony/pics/unicorn.jpg');
 //=> 'file:///Users/pony/pics/unicorn.jpg'
+
+// passing false as second argument will not call path.resolve() on the path
+fileUrl('unicorn.jpg', false);
```

Comment:
Should be an options object, not a [boolean trap](http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html).